gamepanel/gamehome.php

```diff
@@ -188,32 +189,35 @@ function links()
 		{
 			$noticesStatus = 5;
 			$SubmitName = 'Toggle Notices';
+			if (($User->optInFeatures & 0x1) > 0) { $playBeta = '<a href="board.php?gameID='.$this->id.'/beta #gamePanel" >'.l_t('Play Beta').'</a> '; }
```

Comment:
again, it seems like maybe this should be a property of User? like isBeta or something so we don't have to keep making this comparison

Reply:
so they are using this integer bitset to capture if user is opted in to various experimental features. Looks like they did it this way to minimize the effort of adding or removing additional features without DB changing. I thought having different properties for each features would make more sense with having them treated separately in the DB. I'm not familiar with this concept so please let me know if I misinterpreted

Reply:
@hyukcodazen maybe we can abstract this comparison to an easy to call ENUM of some sort. As Steve mentioned, even though this bit comparison works it is difficult to know exactly what it's doing without diving into it.